jobs/rocoto/awips.sh

```diff
@@ -94,14 +83,14 @@ for fhr in $fhrlst; do
             done
             
 	    export fcsthrs=$fhr3
-	    export job="jgfs_awips_f${fcsthrs}_20km_${cyc}"
-	    export DATA="${DATAROOT}/$job"
+	    # export job="jgfs_awips_f${fcsthrs}_20km_${cyc}"
+	    # export DATA="${DATAROOT}/$job"
```

Comment:
are we sure this is not going to bomb?

Reply:
No, but the AWIPS jobs aren't working right now anyway for unrelated reasons. It isn't causing any immediate issues up until that point. But that is why I left them as comments for now instead of just deleting them outright, so we can see how it was supposed to work before when we get a chance to fix the AWIPS job.

Reply:
understood.  thanks for explaining.